inspectit-ocelot-documentation/docs/getting-started/docker-examples.md

```diff
@@ -21,7 +21,7 @@ Uses InfluxData Telegraf for metrics gathering, InfluxDB for metrics storage and
 
 ![Demo scenario using InfluxDB and Zipkin](assets/demo-landscape-influxdb-zipkin.png)
 
-### Demo #2 - InfluxDB and Zipkin
+### Demo #2 - Prometheus,Grafana and jaeger
```

Comment:
Can you capitalize all words except `and` and add a whitespace before grafana :)

`Prometheus, Grafana and Jaeger`

Reply:
Done!